engine/src/main/java/org/pentaho/di/trans/steps/exceloutput/ExcelOutput.java

```diff
@@ -834,31 +817,30 @@ private void setFonts() throws Exception {
         data.headerImageHeight = m.getHeight() * 0.0625;
 
         byte[] imageData = new byte[(int) imageFile.getContent().getSize()];
-        KettleVFS.getInputStream( imageFile ).read( imageData );
+        imageStream = KettleVFS.getInputStream( imageFile );
+        imageStream.read( imageData );
 
         data.headerImage = new WritableImage( 0, 0, data.headerImageWidth, data.headerImageHeight, imageData );
       } catch ( Exception e ) {
         throw new KettleException( e );
       } finally {
-        if ( imageFile != null ) {
+        if ( imageStream != null ) {
```

Comment:
this block became unnecessary since imageFile is closed as AutoClosable, and it closes the inputStream as well